src/Dictionaries/HashedDictionary.cpp

```diff
@@ -1192,9 +1155,14 @@ void registerDictionaryHashed(DictionaryFactory & factory)
         if (shard_load_queue_backlog <= 0)
             throw Exception(ErrorCodes::BAD_ARGUMENTS,"{}: SHARD_LOAD_QUEUE_BACKLOG parameter should be greater then zero", full_name);
 
+        float max_load_factor = static_cast<float>(config.getDouble(config_prefix + dictionary_layout_prefix + ".max_load_factor", 0.5));
+        if (max_load_factor < 0.5 || max_load_factor > 0.99)
```

Comment:
(Ah, I asked about a sanity check for max_load_factor in another comment. Here it is.)